features/git-kill/current_branch/on_feature_branch/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,49 @@
+Feature: git kill: killing the current feature branch without a tracking branch (with open changes and without remote repo)
+
+  (see ../without_tracking_branch/with_open_changes.feature)
+
+
+  Background:
+    Given my repo does not have a remote origin
+    And I have local feature branches named "current-feature" and "other-feature"
+    And the following commits exist in my repository
+      | BRANCH          | LOCATION | MESSAGE                | FILE NAME            | FILE CONTENT            |
+      | other-feature   | local    | other feature commit   | other_feature_file   | other feature content   |
+      | current-feature | local    | current feature commit | current_feature_file | current feature content |
```

Comment:
Sorry, I forgot to mention that you could sort by branch name. This way the current-feature comes first, which I think makes sense here.